components/brave_wallet/renderer/js_ethereum_provider.cc

```diff
@@ -30,6 +30,7 @@
 #include "third_party/blink/public/web/web_console_message.h"
 #include "third_party/blink/public/web/web_local_frame.h"
 #include "url/origin.h"
+#include "third_party/blink/renderer/platform/bindings/script_state.h"
```

Comment:
see below, you can't access anything in platform outside of blink

Reply:
ya this was just prototype to see if I could get it to work. Will do.